README.md

```diff
@@ -115,7 +115,7 @@ with excellent support for Angular apps that use routing.
 
 Here are the test related scripts:
 * `npm test` - compiles, runs and watches the karma unit tests
-* `npm run e2e` - run protractor e2e tests, written in JavaScript (*e2e-spec.js)
+* `npm run e2e` - compiles and run protractor e2e tests, written in JavaScript (*e2e-spec.js)
```

Comment:
Yes, and that means they need to be compiled. However what I actually wanted to say here (and didn't do it very well) is that running the command also compiles your app.